staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -236,6 +238,20 @@ func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionK
 		return nil, err
 	}
 
+	uList := unstructured.UnstructuredList{}
```

Comment:
consider use fake-dynamic-client for testing/faking unstructured objects? https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/client-go/dynamic/fake

Reply:
@yue9944882 I understand the dynamic client would be recommended for that use case, however, the current implementation is removing `kind` and `apiVersion` values. Is that the expected behavior?

Reply:
it is expected, but is not desired. see https://github.com/kubernetes/kubernetes/issues/80609